talk/xmpp/xmppengineimpl.h

```diff
@@ -102,6 +103,27 @@ class XmppEngineImpl : public XmppEngine {
     lang_ = lang;
   }
 
+  //! Sets connection port
+  virtual void SetSocketServer(const std::string& server, int port) {
+    connection_server_ = server;
+    connection_port_ = port;
+
+    std::string hostname = tls_server_hostname_;
+    if (hostname.empty())
+      hostname = this->user_jid_.domain();
+
+    // If not language is specified, the spec says use *
+    std::string lang = lang_;
+    if (lang.length() == 0)
+      lang = "*";
```

Comment:
Hi @dscherba,

In the original code of WebRTC, I saw that they usually didn't use braces in one line if command. You can easily see it in the code.